openmdao/utils/scaffolding_templates/implicit_comp_template

```diff
@@ -0,0 +1,140 @@
+
+
+import numpy as np
+
+from openmdao.core.implicitcomponent import ImplicitComponent
+
+
+class {class_name}(ImplicitComponent):
+    """
+    <class description here>
+
+    Attributes
+    ----------
+    ...
+    """
+
+    def __init__(self, **kwargs):
+        """
+        Intialize this component.
+
+        Parameters
+        ----------
+        **kwargs : dict of keyword arguments
+            Keyword arguments that will be mapped into the Component options.
+        """
+        super({class_name}, self).__init__(**kwargs)
+        # set attributes here...
+
+    def initialize(self):
+        """
+        Declare options.
+        """
+        # self.options.declare('myopt', default=1, types=int, desc='My option.')
+        pass
+
+    def setup(self):
+        """
+        Add inputs and outputs and declare partials.
+        """
+        # self.add_input(name='foo', shape=None, units=None)
+        # self.add_output(name='bar', shape=None, units=None)
+
+        # self.declare_partials(of='bar', wrt='foo', rows=None, cols=None, val=None)
+
+        # # Set up the derivatives.
+        # self.declare_partials(of='bar', wrt='foo', rows=None, cols=None, val=None)
```

Comment:
looks like you have some unintended duplication here, as well as a commented comment

Reply:
fixed